### ISSUE TITLE
Does export to XML include my own messages?

### ISSUE THREAD

**Issue by SamirHafez** [johanw666/Signal-Android#36]
I've imported the generated XML into my SMS App. I've noticed that imported threads contain my messages when the thread was using SMS in Signal (as in Signal managed SMSs). However, if the thread was using the Signal protocol instead, then I only see the messages from my contacts and not my own. Is this expected?

**Comment by johanw666** [comment]
No, that should not have happened. I'll have to check and see what is going on here.
